src/xd/xd68/xd68.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "XD68",
+  "vendorId": "0x7844",
+  "productId": "0x6868",
+  "lighting": "qmk_backlight_rgblight",
+  "matrix": {"rows": 5, "cols": 15},
+  "layouts": {
+    "labels": [
+        ["Enter Key", "ANSI", "ISO"],
+        "Split Left Shift"
+    ],
+    "keymap": [
+      [
+        {
+          "x": 2.5,
+          "c": "#777777"
+        },
+        "0,0",
+        {
+          "c": "#cccccc"
+        },
+        "0,1",
+        "0,2",
+        "0,3",
+        "0,4",
+        "0,5",
+        "0,6",
+        "0,7",
+        "0,8",
+        "0,9",
+        "0,10",
+        "0,11",
+        "0,12",
+        {
+          "c": "#aaaaaa",
+          "w": 2
+        },
+        "0,13",
+        {
+          "c": "#cccccc"
+        },
+        "0,14"
+      ],
+      [
+        {
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.5
+        },
+        "1,0",
+        {
+          "c": "#cccccc"
+        },
+        "1,1",
+        "1,2",
+        "1,3",
+        "1,4",
+        "1,5",
+        "1,6",
+        "1,7",
+        "1,8",
+        "1,9",
+        "1,10",
+        "1,11",
+        "1,12",
+        {
+          "w": 1.5
+        },
+        "1,13\n\n\n0,0",
+        "1,14",
+        {
+          "x": 1.5,
+          "c": "#777777",
+          "w": 1.25,
+          "h": 2,
+          "w2": 1.5,
+          "h2": 1,
+          "x2": -0.25
+        },
+        "2,13\n\n\n0,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "2,0",
+        {
+          "c": "#cccccc"
+        },
+        "2,1",
+        "2,2",
+        "2,3",
+        "2,4",
+        "2,5",
+        "2,6",
+        "2,7",
+        "2,8",
+        "2,9",
+        "2,10",
+        "2,11",
+        {
+          "c": "#777777",
+          "w": 2.25
+        },
+        "2,13\n\n\n0,0",
+        {
+          "c": "#cccccc"
+        },
+        "2,14",
+        {
+          "x": 0.5
+        },
+        "2,12\n\n\n0,1"

```

Comment:
Recommendation to check this ISO key.

Reply:
Ah, I think I get what you're meaning here, misunderstanding on my part (or naïve assumption, rather) - I have the extra key handled in the VIA layout, but since there is no corresponding QMK key in the layout, the key wouldn't be mapped correctly, is that right?  Appreciate the extra look - I'll check this and submit a correction.

Reply:
Yes, exactly. I would recommend checking everything to make sure there is a corresponding key in QMK to match the VIA .json file. It may just require a QMK submission.  Cheers.

Reply:
Testing on the actual hardware - the key...works?  I'm not entirely sure how.  I flashed the default ansi QMK layout, and the key has no response - expected.  Flash via, using this json file - and the key works as expected.  I imagine it'd be best to have explicit representation of the key in QMK - but is VIA magically filling in the gaps here?

Reply:
It will probably work but it might be better to have an explicit representation in QMK as you say. At least I hope this helps you decide if you want to define the key in QMK when someone chooses to use it.